data/sheragi/archaeology missions.txt

```diff
@@ -212,19 +212,37 @@ mission "Deep Archaeology 5"
 		conversation
 			`You breathe a sigh of relief as you land safely in the spaceport on <planet>. "Okay, what was that all about?" you ask.`
 			`	"I honestly don't know," says Albert. "I can tell you what I suspect, though. The history books don't actually say when the Deep was first settled. The first documented expedition to the Deep was in the 24th century, and they found several human colonies already established in the region, claiming to date back to the very dawn of space flight. But I think the first settlers were brought there long before that. Brought there on alien ships."`
+			branch "pug"
+				or
+					has "FWC Pug 2: offered"
+					has "FW Pug 3: offered"
 			choice
 				`	"For what purpose?"`
 					goto purpose
 				`	"What aliens? The Quarg?"`
 					goto aliens
+			label pug
+			choice
+				`	"For what purpose?"`
+					goto purpose
+				`	"Wait, are you talking about the Pug?"`
```

Comment:
```suggestion
				`	"Do you think it was the Pug?"`
```
To me, the `Wait, are you talking about` feels like it implies that `<first>` thinks Albert's holding back info from them, which I think isn't what `<first>` would think in this situation. It really doesn't matter a whole lot though - your choice.